src/pages/search/project_search/ProjectSearch.tsx

```diff
@@ -24,14 +25,22 @@ export const ProjectSearch = () => {
   if (!projectSearchQueryParams.get(SearchParam.Results)) {
     projectSearchQueryParams.set(SearchParam.Results, '10');
   }
+  if (!projectSearchQueryParams.get(SearchParam.Page)) {
+    projectSearchQueryParams.set(SearchParam.Page, '1');
+  }
 
-  const queryParams = projectSearchQueryParams.toString();
+  const rowsPerPage = Number(projectSearchQueryParams.get(SearchParam.Results));
+  const page = Number(projectSearchQueryParams.get(SearchParam.Page));
+  const titleQuery = projectSearchQueryParams.get(SearchParam.Query);
```

Comment:
Er dette enklere? Da holder det også med `{ query }` på linje 38
```suggestion
  const query = projectSearchQueryParams.get(SearchParam.Query) ?? '';
```